forem/forem
# Improve devcontainer

## What type of PR is this? (check all applicable)

## Description
Collection of improvements to devcontainer/codespace.
- setup test database on boot. Previous it does not because of strict 
- Add more sensible plugins
- Add a container specific community name so it's more obvious which server is running.

## Related Tickets & Documents
resolves https://github.com/forem/forem-internal-eng/issues/656
## QA Instructions, Screenshots, Recordings
You are welcome to open up a codespace of this instance to make sure things are working normally and by running a test. It may require a bit of time though as there's not prebuild ready yet.

## Added tests?

## Added to documentation?

## [optional] Are there any post deployment tasks we need to perform?
n/a
